### DISCUSSION TITLE
Send files to an external SFTP based on an action/event

### DISCUSSION THREAD

**Discussion by josh383451** [drakkan/sftpgo#1328] [Q&A]
Hi,

Please could someone tell me how I can upload a file to an external SFTP server based on an action? My scenario is below.

Files are generated in local filesystem of /opt/tmp > The file I want to upload externally has a specific name in the file, for example \*ABC\*.txt > Send that file to an external SFTP server.

Apologies if this has been asked before. This is a great product and would like to explore it further.

Thank you and Regards.

**Comment by JonBendtsen** [comment]
what ever generates those files might as well just upload directly to the external SFTP server?

**Reply by josh383451** [reply]
Believe me, if it were that easy then yes I would be doing that. Our application doesn't have that functionality (yet).

**Reply by JonBendtsen** [reply]
Okay, then what if you made a cron job that runs regularly and uploads the files? Or use ionotify to upload as soon as the file is closed by your other application?

**Reply by josh383451** [reply]
That's what we're currently doing but something like this with a GUI and easier user management, provisioning, and automation will be a better solution.

**Reply by JonBendtsen** [reply]
I do not think that SFTPgo is the tool you are looking for
